cfme/tests/control/test_bugs.py

```diff
@@ -46,6 +49,11 @@ def alert_collection(appliance):
     return appliance.collections.alerts
 
 
+@pytest.fixture(scope="module")
+def alert_profile_collection(appliance):
```

Comment:
Please don't make this a fixture, as there's no setup happening here.

I realize that you're just repeating an existing pattern in this test module, but its not a good one. All collections can be easily fetched from the appliance directly. I would allow for collection use in a fixture that is applying a filter to that collection for all tests.

Reply:
Agreed, I will change this.